tests/test_utils.py

```diff
@@ -0,0 +1,128 @@
+# This file is part of summit_utils.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Test cases for utils."""
+
+import copy
+import itertools
+import unittest
+
+import astropy.time
+import astropy.units as u
+import lsst.afw.image as afwImage
+import lsst.geom as geom
+import lsst.utils.tests
+import numpy as np
+from astro_metadata_translator import makeObservationInfo
+from lsst.obs.base import createInitialSkyWcsFromBoresight
+from lsst.obs.base.makeRawVisitInfoViaObsInfo import MakeRawVisitInfoViaObsInfo
+from lsst.obs.lsst.latiss import Latiss
+from lsst.summit.utils.utils import getExpPositionOffset
+
+
+class ExpSkyPositionOffsetTestCase(lsst.utils.tests.TestCase):
+    """A test case for testing sky position offsets for exposures."""
+
+    def setUp(self):
+        camera = Latiss.getCamera()
+        self.assertTrue(len(camera) == 1)
+        self.detector = camera[0]
+
+        self.viMaker = MakeRawVisitInfoViaObsInfo()
+        self.mi = afwImage.maskedImage.MaskedImageF(0, 0)
+        self.baseHeader = dict(boresight_airmass=1.5,
```

Comment:
oh, this is a dict pretending to be an ObservationInfo? You need to create an astropy.coordinates.EarthLocation object and attach it to `location` in here.

Reply:
Ah, great, thanks!